src/modules/did/did.processor.ts

```diff
@@ -17,8 +16,7 @@ export class DIDProcessor {
   public async processDIDDocumentUpsert(job: Job<string>) {
```

Comment:
I think ideally the naming would be consistent, so if the didService method being called is `addCacheDocument`, then maybe this method should be called `processDIDDocumentAddition`

Reply:
👍